src/mame/skeleton/evolution_handheld.cpp

```diff
@@ -0,0 +1,82 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+
+// TODO: identify the CPU type! - there are vectors(?) near the start
+
+#include "emu.h"
+
+#include "speaker.h"
+#include "screen.h"
+
+#include "cpu/evolution/evo.h"
```

Comment:
Please put headers from src/emu after headers from src/devices, and keep the headers from src/emu in alphabetical order.